src/services/logger/Logger.ts

```diff
@@ -0,0 +1,26 @@
+import { datadogLogs } from '@datadog/browser-logs'
+
+import { kumaApi } from '@/api/kumaApi'
+import type Env from '@/services/env/Env'
+
+export default class Logger {
```

Comment:
**Suggestion**: Can you please rename this to something more specific? We don’t use Datadog as a general purpose logging utility, but only in a few places.

Reply:
Here's an example of one place where we use it (but not as yet via the service):

https://github.com/kumahq/kuma-gui/blob/9c37737efeb8b8054aa9a190b1e6bc2813345e77/src/app/common/PaginationWidget.vue#L59-L66

I partly imagined at some point in the future if we wanted to, we could change the:

```javascript
datadogLogs.logger.info(datadogLogEvents.PAGINATION_NEXT_BUTTON_CLICKED) 
```

to

```javascript
const { logger, loggerEvents } = useLogger()
logger.info(loggerEvents.PAGINATION_NEXT_BUTTON_CLICKED) 
```

or similar, which we could switch out during development for a local logger of some sort.

Apart from `Logger` do you have another name in mind? We could call this one `RemoteLogger` perhaps?

Reply:
I would much rather call this DatadogLogger. It should be obvious when we’re logging something to an external place *and* what that external place is.

Reply:
K I changed this to DatadogLogger